README.md

```diff
@@ -54,3 +54,8 @@ Please follow this link to learn about backend service.
 
 https://github.com/mojaloop/ml-testing-toolkit/blob/master/README.md
 
+-------
+
+a. auditing-dependencies: https://github.com/mojaloop/central-ledger/blob/master/README.md#auditing-dependencies
```

Comment:
What I meant by this is that you should copy and paste those sections into this document, not reference as you have done.

Reply:
Sorry for being so dumb :). Done